### PR TITLE
removed python2.7 fixes #132

### LINKED ISSUE #132: Remove Python2.7 installs from setup.sh
We should avoid installing the old libraries if we aren't utilizing them.

Please remove them so we can test.

### PR DESCRIPTION
**Pull request by gacosta1**

### CODE REVIEW

File: Software/setup.sh [hunk 1]
```diff
@@ -4,14 +4,6 @@
...
-#install python setup tools::
-sudo apt-get --assume-yes install python-setuptools
-
-sudo apt-get --assume-yes install python
-
-#install pip for python 2::
-sudo apt-get --assume-yes install python-pip
-
 #Now install the Dev Kit::
 sudo apt-get --assume-yes install python-dev python-pip gcc
```
Comment (ophilli) [inline Software/setup.sh:8]: Do we need to specify python3-dev python3-pip?

File: Software/setup.sh [hunk 2]
```diff
@@ -25,17 +17,20 @@ sudo apt-get --assume-yes install python3-setuptools
 
 sudo easy_install3 pip
```
Comment (ophilli) [inline Software/setup.sh:18]: Shouldn't python3 install pip?

Comment (ophilli) [reply Software/setup.sh:18]: Why do we install pip here? & on line 24

File: Software/setup.sh [hunk 3]
```diff
@@ -25,17 +17,20 @@ sudo apt-get --assume-yes install python3-setuptools
...
 
+#install python3
 sudo apt-get --assume-yes install python3
```
Comment (ophilli) [inline Software/setup.sh:21]: We just installed this on line 13

File: Software/setup.sh [hunk 4]
```diff
@@ -25,17 +17,20 @@ sudo apt-get --assume-yes install python3-setuptools
...
 
+#install python3 pip tool
 sudo apt-get --assume-yes install python3-pip
 
+#install the sql 
 sudo pip3 install pymysql
```
Comment (ophilli) [inline Software/setup.sh:27]: These pip installs should be moved out to requirements.txt

**Review by ophilli — APPROVED** [review]
It's better than before - but still a lot we can improve!

File: Software/setup.sh (hunk 1 above)
Comment (gacosta1) [reply Software/setup.sh:8]: yeah you right

File: Software/setup.sh (hunk 2 above)
Comment (gacosta1) [reply Software/setup.sh:18]: I'm honestly not sure why we I added sudo easy_install3 pip. Maybe I needed that to run the program back then. I could be wrong, only way to test is to reset the pi and run the setup then run the program

File: Software/setup.sh (hunk 3 above)
Comment (gacosta1) [reply Software/setup.sh:21]: But python and python3 are different! unless installing python will automatically install python3, but I don't think so?

File: Software/setup.sh (hunk 4 above)
Comment (gacosta1) [reply Software/setup.sh:27]: An assignment for @r3cebarnett or @gmross

File: Software/setup.sh (hunk 3 above)
Comment (ophilli) [reply Software/setup.sh:21]: @gacosta1 What I'm seeing on line 13 is that we run `sudo apt-get --assume-yes install python3`

File: Software/setup.sh (hunk 4 above)
Comment (ophilli) [reply Software/setup.sh:27]: @gacosta1 please review #98

**Pull request merged**